src/settings/emaildomain/AddEmailAddressesPage.ts

```diff
@@ -184,6 +184,7 @@ export class AddEmailAddressesPageAttrs implements WizardPageAttrs<AddDomainData
 					return false
 				} else if (e instanceof UpgradeRequiredError) {
 					await showPlanUpgradeRequiredDialog(e.plans, e.message)
+					return false
```

Comment:
that's what you get for not having an `else` case huh

Reply:
I mean, the else case would be throwing the error, so...

Reply:
yeah but you wouldn't just fallthrough to it